src/cmd/utilities.rs

```diff
@@ -58,36 +58,17 @@ where
     cmd
 }
 
-pub fn exec<P>(binary: P, args: Vec<&str>) -> Result<(), SimpleError>
+pub fn exec<P>(binary: P, args: Vec<&str>, envs: Option<Vec<(&str, &str)>>) -> Result<(), SimpleError>
```

Comment:
Change from Option<Vec> to Vec.
Pass am empty vector if you don't want env variables

Reply:
I agree with you and tought about it as well but felt None is move explicit and easier than vec!["", ""]